openshift_images/using_images/using-s21-images.adoc

```diff
@@ -0,0 +1,30 @@
+[id="using-s21-images"]
+= Source-to-image
+include::modules/common-attributes.adoc[]
+:context: using-s21-images
+
+toc::[]
+
+You can use the link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html-single/using_red_hat_software_collections_container_images/index[Red Hat Software Collections] images as a foundation for applications that rely on specific runtime environments such as Node.js, Perl, or Python. Special versions of some of these runtime base images are referred to as Source-to-Image (S2I) images. With S2I images, you can insert your code into a base image environment that is ready to run that code.
+
+S2I images include:
+
+* Java
+* Node.js
+* Perl
+* PHP
+* Python
+* Ruby
+
+S2I images are available for you to use directly from the {product-title} web UI by selecting *Catalog* -> *Developer Catalog*.
+
+S2I images are also available though the Cluster Samples Operator.
+
+include::modules/images-s2i-build-process-overview.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
Vikram said to not add the PV2 metadata yet, but I would add an explicit anchor.